smelli/classes.py

```diff
@@ -229,82 +295,41 @@ def log_likelihood_global(self):
-                    info[obs]['pull'] = pull(-2 * (ll - ll_central), dof=pull_dof)
-                    ll_sm = m.get_logprobability_single(obs, pred_sm[obs])
-                    s = 1 if ll > ll_sm else -1
-                    info[obs]['delta pull'] = -s * pull(-2 * (ll - ll_sm), dof=pull_dof)
+                    info[obs]['pull'] = pull(-2 * (ll - ll_central), dof=1)
+                    info[obs]['delta_ll'] = (ll - ll_sm)[0]
             for lh_name, lh in llh.likelihoods.items():
                 # loop over "normal" likelihoods
                 ml = lh.measurement_likelihood
                 pred = ml.get_predictions_par(llh.par_dict, self.w)
-                pred_sm = ml.get_predictions_par(llh.par_dict, flavio.WilsonCoefficients())
-                inspire_dict = self._get_inspire_dict(lh.observables, ml)
                 for i, obs in enumerate(lh.observables):
-                    obs_dict = flavio.Observable.argument_format(obs, 'dict')
-                    obs_name = obs_dict.pop('name')
-                    with warnings.catch_warnings():
-                        warnings.simplefilter("ignore")
-                        p_comb = flavio.combine_measurements(
-                            obs_name,
-                            include_measurements=ml.get_measurements,
-                            **obs_dict)
-                    info[obs]['experiment'] = p_comb.central_value
-                    info[obs]['exp. unc.'] = max(p_comb.error_left, p_comb.error_right)
-                    info[obs]['exp. PDF'] = p_comb
-                    info[obs]['inspire'] = sorted(set(inspire_dict[obs]))
                     info[obs]['theory'] = pred[obs]
-                    info[obs]['th. unc.'] = 0
-                    info[obs]['lh_name'] = lh_name
-                    info[obs]['name'] = obs if isinstance(obs, str) else obs[0]
-                    ll = p_comb.logpdf([pred[obs]]) - p_comb.logpdf([p_comb.central_value])
-                    info[obs]['pull'] = pull(-2 * ll, dof=pull_dof)
-                    delta_ll = p_comb.logpdf([pred[obs]]) - p_comb.logpdf([pred_sm[obs]])
-                    s = 1 if delta_ll > 0 else -1  # sign of delta_ll
-                    info[obs]['delta pull'] = -s * pull(-2 * abs(delta_ll), dof=pull_dof)
+                    ll_central = info[obs]['ll_central']
+                    ll_sm = info[obs]['ll_sm']
+                    p_comb = info[obs]['exp. PDF']
+                    ll = p_comb.logpdf([pred[obs]])
+                    info[obs]['pull'] = pull(-2 * (ll - ll_central), dof=1)
+                    info[obs]['delta_ll'] = (ll - ll_sm)[0]
             self._obstable_tree_cache = info
         return self._obstable_tree_cache
 
-    def obstable(self, min_pull=0):
+    def obstable(self, min_pull=0, sort_by='pull', min_val=None, max_val=None,
+                 ascending = None):
         r"""Return a pandas data frame with the central values and uncertainties
```

Comment:
no spaces around = :)

Reply:
Done in https://github.com/smelli/smelli/pull/1/commits/c44bac3e253812c62a818a0b52bc80c9a56a402f.